docs/source/learn/bazel.rst

```diff
@@ -0,0 +1,259 @@
+Bazel in Pytorch/XLA
```

Comment:
Let's keep developer/contributor documentation like this separate from user documentation.

Reply:
Sounds good. I read through that doc and was unclear about what the purpose of the doc was. It talks about bazel, but doesn't talk about how it's used. I did a little digging and found that its used to build software and it's dependcies. I assume that the build process for Pytorch and/or Pytorch/XLA uses bazel. When would customers need to know this information?

Reply:
Bazel is used for building PyTorch/XLA itself. This page is only useful for contributors who are updating our build process, which is mostly just people on this team.

Some other docs that are only relevant to contributors and not users:

- OpenXLA (we can delete this IMO)
- Codegen migration Guide
- OP Lowering Guide
- Custom Hardware Plugins

Maybe we can leave CONTRIBUTING in `docs`, and create a separate folder and TOC group like "Contribute to PyTorch/XLA"?

Reply:
Done